chains/evm/contracts/offRamp/OffRamp.sol

```diff
@@ -245,11 +249,11 @@ contract OffRamp is ITypeAndVersion, Ownable2StepMsgSender {
     uint16 maxReturnBytes = Internal.MAX_RET_BYTES;
 
     uint256 gasLeft = gasleft();
-    if (gasLeft <= MAX_GAS_BUFFER_TO_UPDATE_STATE) {
+    if (gasLeft <= s_maxGasBufferToUpdateState) {
```

Comment:
caching this like
`uint256 gasBuffer = s_maxGasBufferToUpdateState;`

saves 136 gas

Reply:
Oh nice catch, fixed! [4be9538](https://github.com/smartcontractkit/chainlink-ccip/pull/1511/commits/4be953888d1cebb69ffab0e723cb3075c362259f)